src/main/java/Homework2/CanRunnig.java

```diff
@@ -0,0 +1,5 @@
+package Homework2;
+
+public interface CanRunnig {
+    abstract String howCanRunVehicle();
```

Comment:
abstract modifier is redundant as methods in interfaces are abstract and public by default